test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -13575,6 +13577,22 @@ def create_script(self, body=None, uid=None, gid=None, hostname=None):
         if isinstance(body, list):
             body = '\n'.join(body)
 
+        if self.platform == 'cray' or self.platform == 'craysim':
+            body = body.split("\n")
+            for line in body:
+                if line.startswith("#PBS") and "select=" in line:
+                    line_index = body.index(line)
+                    line_arr = line.split(" ")
+                    for j in line_arr:
+                        if j.startswith("select="):
+                            select_index = line_arr.index(j)
+                            sel_str = j.replace("select=", "")
+                            sel_str = self.add_cray_vntype(select=sel_str)
+                            j = "select=" + sel_str
+                            line_arr[select_index] = j
```

Comment:
Don't reuse the iteration variable.  Just say line_arr[select_index] = "select=" + sel_str